Formula/gromacs.rb

```diff
@@ -50,11 +51,15 @@ def install
 
     inreplace "src/gromacs/gromacs-config.cmake.cmakein", "@GROMACS_CXX_COMPILER@", cxx
 
-    mkdir "build" do
-      system "cmake", "..", *std_cmake_args, "-DGROMACS_CXX_COMPILER=#{cxx}",
-                                             "-DGMX_VERSION_STRING_OF_FORK=#{tap.user}"
-      system "make", "install"
-    end
+    args = %W[
+      -DGROMACS_CXX_COMPILER=#{cxx}
+      -DGMX_VERSION_STRING_OF_FORK=#{tap.user}
+    ]
+    # Force SSE2 for compatibility. The next available option is SSE4.1
+    args << "-DGMX_SIMD=SSE2" if OS.linux? && Hardware::CPU.intel? && build.stable? && build.bottle?
```

Comment:
Default is `AUTO` and will optimize to host hardware.

Options are: `None SSE2 SSE4.1 AVX_128_FMA AVX_256 AVX2_256 AVX2_128 AVX_512 AVX_512_KNL ARM_NEON_ASIMD ARM_SVE IBM_VSX`

Not sure if we want to make macOS builds compatible/portable

Reply:
If needed on macOS, then maybe following (not sure if `sse4` or `sse42` is accurate):
```suggestion
    simd = MacOS.version.requires_sse4? ? "SSE4.1" : "SSE2"
    args << "-DGMX_SIMD=#{simd}" if Hardware::CPU.intel? && build.stable? && build.bottle?
```

Reply:
It should ideally cover macOS too IMO.